.github/workflows/test-suite-desktop-e2e.yml

```
@@ -21,14 +24,26 @@ on:
   schedule:
     - cron: "0 0 * * *"
   workflow_dispatch:
+  issue_comment:
+    types:
+      - created
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)